adr/adr-guidelines.md

```diff
@@ -0,0 +1,26 @@
+# Architectural Decision Record Formatting Checklist
+Below is a formatting checklist with some guidelines that serves to help keep our ADRs looking clean and uniform. 
+
+1. See [adr/001-architecture-decision-records.md](adr/001-architecture-decision-records.md) for template and section guidelines/requirements.
+
+
+2. Review ‘Context’ section for grammar, typos, and clarity. ** **Keep in mind -** ** readers may have less inside knowledge or experience with the project when reading these documents.
```

Comment:
I think the formatting is maybe off here? 
![image](https://github.com/user-attachments/assets/8ff1ad05-827e-4cd1-b7e2-59e3c8979a07)

Reply:
Oh no, I just checked - I wanted the stars and bold in Markdown so it displays as expected in preview mode.